spock-specs/src/test/groovy/org/spockframework/mock/runtime/mockito/MockitoInlineMockMakerSpec.groovy

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.mock.runtime.mockito
+
+import org.mockito.MockMakers
+import org.mockito.Mockito
+import org.mockito.exceptions.base.MockitoException
+import org.spockframework.mock.CannotCreateMockException
+import org.spockframework.mock.MockUtil
+import org.spockframework.runtime.GroovyRuntimeUtil
+import spock.lang.Issue
+import spock.lang.PendingFeature
+import spock.lang.Requires
+import spock.lang.Specification
+import spock.mock.DetachedMockFactory
+
+import java.lang.reflect.Proxy
+import java.util.concurrent.Callable
+
+import static spock.mock.MockMakers.mockitoInline
+
+@SuppressWarnings("GroovyAssignabilityCheck")
+class MockitoInlineMockMakerSpec extends Specification {
+  private static final String ID = "mockito-inline"
+
+  def "Verify simple ID and IMockMakerSettings"() {
+    expect:
+    mockitoInline.mockMakerId.toString() == ID
+    mockitoInline.toString() == ID
+  }
+
+  def "Verify ID and IMockMakerSettings with Mockito settings"() {
+    when:
+    def set = mockitoInline {}
+    then:
```

Comment:
:x: Add a newline before the block labels when starting a new block, to increase readability.